books/user/src/platform.md

```diff
@@ -0,0 +1,46 @@
+# Platform support
+
+Support for different platforms ("targets") are organized into three tiers,
+each with a different set of guarantees. Targets are identified by the
+[Rust "target triple"](https://doc.rust-lang.org/rustc/platform-support.html)
+which is the string used when compiling `cuprated`.
+
+| Attribute           | Tier 1 | Tier 2            | Tier 3 |
+|---------------------|--------|-------------------|--------|
+| Official builds     | 🟢     | 🟢                | 🔴
+| Guaranteed to build | 🟢     | 🟢                | 🟡
+| Automated testing   | 🟢     | 🟡 (some targets) | 🔴
+| Manual testing      | 🟢     | 🟡 (sometimes)    | 🔴
```

Comment:
Basic platform support rules for `v0.0.1`.

`Tier 1` are ones I can currently test; I don't have a reliable Windows machine setup yet so `Tier 2` for now. Official builds for `v0.0.1` will be:

- `x64 Linux`
- `ARM64 Linux`
- `x64 macOS`
- `ARM64 macOS`
- `x64 Windows`